test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4176,34 +4160,6 @@ BOOST_AUTO_TEST_CASE(index_access_with_type_conversion)
 	BOOST_CHECK(callContractFunction("f(uint256)", u256(0x101)).size() == 256 * 32);
 }
 
-BOOST_AUTO_TEST_CASE(failed_create)
-{
-	char const* sourceCode = R"(
-		contract D { constructor() payable {} }
-		contract C {
-			uint public x;
-			constructor() payable {}
-			function f(uint amount) public returns (D) {
-				x++;
-				return (new D){value: amount}();
-			}
-			function stack(uint depth) public returns (address) {
-				if (depth < 1024)
-					return this.stack(depth - 1);
-				else
-					return address(f(0));
-			}
-		}
-	)";
-	compileAndRun(sourceCode, 20, "C");
-	BOOST_CHECK(callContractFunction("f(uint256)", 20) != encodeArgs(u256(0)));
-	ABI_CHECK(callContractFunction("x()"), encodeArgs(u256(1)));
-	ABI_CHECK(callContractFunction("f(uint256)", 20), encodeArgs());
```

Comment:
Is `encodeArgs()` equivalent to failure?

Reply:
the old tests were not able to distinguish failure from success.